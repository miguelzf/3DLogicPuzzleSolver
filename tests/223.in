- - - - - -
- - - - C -
- - P - - -
- - - - - -
- - R - - -
- - - - - R

- - - - - X
- C - - - -
- - - - B G
- - P - - -
- - - - - -
- - - - - -

- - - - - -
- - B - - Y
- - - - - -
- - - - - G
- - - - - -
- - - Y - -

