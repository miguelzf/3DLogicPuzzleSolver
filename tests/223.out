
Solution:
y  y  y  y  y  y  
y  c  c  c  C  g  
y  c  P  p  g  g  
y  c  -  p  g  -  
y  c  R  p  g  -  
y  c  r  p  g  R  
================
y  c  r  p  g  X  ||  r  r  r  r  g  y  
y  C  r  p  g  g  ||  -  -  B  r  g  Y  
y  r  r  p  B  G  ||  -  -  b  r  g  -  
y  r  P  p  b  b  ||  b  b  b  r  g  G  
y  r  r  r  r  r  ||  r  r  r  r  -  -  
y  y  y  y  y  y  ||  y  y  y  Y  -  -  
